src/patterns/DetailList/DetailList.tsx

```diff
@@ -0,0 +1,64 @@
+import { ComponentProps, forwardRef } from 'react';
+import { getCommonProps } from '../../utils';
+import classnames from 'classnames';
+import { DetailComponent } from '../../components/Detail';
+
+// You'll need to change the ComponentProps<"htmlelementname"> to match the top-level element of your component
+export interface DetailListProps extends ComponentProps<'div'> {
+  /**
+   * Determines whether each Details' label and value are aligned in columns or justified
+   */
+  alignment: 'columns' | 'justified';
+  /**
+   * The Detail components to render
+   */
+  children: DetailComponent[];
+  /**
+   * Whether to render separators between each Detail component
+   */
+  hasSeparators: boolean;
+}
+/**
+ * ## Overview
+ *
+ * A container for displaying a list of [Detail](/docs/components-detail--overview) components. The alignment prop determines whether each Detail's label and value are aligned in columns or justified. Separators can also be added between each Detail component.
+ *
+ * [Figma Link](Add Figma URL here)
```

Comment:
It would be good to point to where this is used in the Figma @aahill50

Reply:
Fixed